src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/RenderOutputParams/index.tsx

```diff
@@ -45,4 +62,4 @@ const RenderOutputParams = () => {
   ) : null
 }
 
-export default RenderOutputParams
+export default withStyles(styles as any)(RenderOutputParams)
```

Comment:
Let's use hook api: https://material-ui.com/styles/basics/#hook-api

Also a reminder: we're trying to get rid of `any` types and adding it for new things is not welcome